hazel.bzl

```diff
@@ -193,7 +193,7 @@ def hazel_custom_package_hackage(
   )
   http_archive(
     name = hazel_workspace(package_name),
-    build_file = "//third_party/haskell:BUILD.{0}".format(package_name),
+    build_file = "@ai_formation_hazel//third_party/haskell:BUILD.{0}".format(package_name),
```

Comment:
I had an issue here as well, using `//` seemed to default to the project I was using Hazel in, not Hazel itself. This seemed to work for me though. I think this was mentioned as an issue in https://github.com/FormationAI/hazel/pull/54

Reply:
This is intentional - Hazel is not intended to be the repository for custom BUILD files, your repository is.

Reply:
We have discussed this before [here](https://github.com/FormationAI/hazel/issues/48) - maybe a documentation fix is in order